test/ngAnimate/integrationSpec.js

```diff
@@ -756,5 +756,38 @@ describe('ngAnimate integration tests', function() {
         expect(child.attr('style')).toContain('50px');
       });
     });
+
+
+    it('should execute the enter animation on a <form> with ngIf that has a <input> of type="email" that is required', function() {
```

Comment:
Could you describe the underlying conditions.
I assume this is not specific to `form`/`ngIf`/`email`/`required`, but some combination of `enter` animation with `addClass`/`removeClass` animations (such as those done by validators). But what exactly ?